enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -23,13 +23,33 @@ configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
 earlier version than 3.14.0 some manual steps must be taken. Links to these
 are provided at each stage of installation and setup that follows.
 
+* [Requirements][Federated Reporting#Requirements]
 * [Installation][Federated Reporting#Installation]
 * [Setup][Federated Reporting#Setup]
 * [Operation][Federated Reporting#Operation]
 * [API Setup][Federated Reporting#API Setup]
 * [Disable Feeder][Federated Reporting#Disable Feeder]
 * [Uninstall][Federated Reporting#Uninstall]
 
+## Requirements ##
+
+If your hub will have SELinux enabled you must install the `semanage` command.
+This allows Federated Reporting policy to manage the trust between the superhub and
+feeder hubs.
+
+Add the `cfengine_mp_fr_dependencies_auto_install` to your augments file to allow
+federation policy to ensure that `semanage` is installed.
+
+```json
+{
+  "classes": {
+    "cfengine_mp_fr_dependencies_auto_install"
+  }
+}
+```
+
+See the [federation.cf semanage_installed](https://github.com/cfengine/masterfiles/blob/master/cfe_internal/enterprise/federation/federation.cf#L144) bundle for details on which packages are used for various distributions.
```

Comment:
Mmmmm, I think we should generally link to the MPF rendered docs instead of the masterfiles repository. I see that we are lacking a federation.cf page https://docs.cfengine.com/docs/master/reference-masterfiles-policy-framework.html want to add it or shall I?

Easily this line number will change, so perhaps use a GitHub search query like
https://github.com/cfengine/masterfiles/search?q=%22bundle+agent+semanage_installed%22&unscoped_q=%22bundle+agent+semanage_installed%22

But, this will only search the default (master_ branch). So it could be misleading for somone reading 3.15.x docs when 3.20 is out.

Reply:
yes, I was considering that the line number would change but didn't know about search query. :+1: I will amend.

Can you point me to the doc that shows me how to add it?

Reply:
Here is an example: https://github.com/cfengine/documentation/blob/master/reference/masterfiles-policy-framework/cfe_internal-core-watchdog-watchdog.markdown

Reply:
Ok. I will do it. Thanks.

Reply:
With the addition of the federation.cf, then you should be able to change this link target to cfe_internal/enterprise/federation/federation.cf#semanage_installed